JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1145,4 +1186,90 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(SEK)
 }
 
-# </editor-fold> HELPER FUNCTIONS BLOCK
\ No newline at end of file
+.descriptivesQQPlot <- function(dataset, options,  qqvar, levelName=NULL) {
+
+  if (options$descriptivesQQPlot && is.null(options$variables))
+    return(list(result=NULL))
+
+  descriptivesQQPlot <- list()
+
+  if (!is.null(qqvar)) {
+
+    #to put a subtitle if there is a split
+    if(!is.null(levelName))
+      descriptivesQQPlot$title <- paste0(options$splitby, ": ", levelName)
```

Comment:
The other plots (e.g., distribution) simply use the level of the grouping variable as the title, to be consistent I think it's good to do that here as well.